.ci/docker/common/install_rocm_magma.sh

```diff
@@ -1,50 +1,28 @@
-#!/bin/bash
-# Script used in CI and CD pipeline
+#!/usr/bin/env bash
+# Script used only in CD pipeline
 
-set -ex
+set -eou pipefail
 
-# Magma build scripts need `python`
-ln -sf /usr/bin/python3 /usr/bin/python
+function do_install() {
+    rocm_version=$1
+    rocm_version_nodot=${1//./}
 
-ID=$(grep -oP '(?<=^ID=).+' /etc/os-release | tr -d '"')
-case "$ID" in
-  almalinux)
-    yum install -y gcc-gfortran
-    ;;
-  *)
-    echo "No preinstalls to build magma..."
-    ;;
-esac
+    # Version 2.7.2 + ROCm related updates
+    MAGMA_VERSION=a1625ff4d9bc362906bd01f805dbbe12612953f6
```

Comment:
It feels wrong to hardcode it here, we better keep all pins in .ci/docker/commit_pin, will land a followup PR